static/app/constants/eventErrors.tsx

```diff
@@ -11,3 +11,7 @@ export enum JavascriptProcessingErrors {
   JS_TOO_LARGE = 'js_too_large',
   JS_FETCH_TIMEOUT = 'js_fetch_timeout',
 }
+
+export enum CocoaProcessingErrors {
+  COCOA_INVALID_DATA = 'invalid_data',
```

Comment:
We would like to only hide it for `contexts.trace.sampled` below Cocoa SDK version `8.7.4` if possible. I don't think hiding all `invalid_data` errors makes sense.